internal/pkg/plugininstaller/util/k8s.go

```diff
@@ -0,0 +1,92 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/devstream-io/devstream/pkg/util/k8s"
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+func CheckAllDeployAndServiceReady(namespace string, deployList []string) error {
+	kubeClient, err := k8s.NewClient()
+	if err != nil {
+		return err
+	}
+
+	// check if all deployments are ready
+	for _, d := range deployList {
+		dp, err := kubeClient.GetDeployment(namespace, d)
+		if err != nil {
+			return err
+		}
+
+		if kubeClient.IsDeploymentReady(dp) {
+			log.Infof("The deployment %s is ready.", dp.Name)
+			continue
+		} else {
```

Comment:
Do not need `else`

Reply:
It might be clearer to write the logic this way:
```go
if !kubeClient.IsDeploymentReady(dp) {
	log.Infof("The deployment %s is not ready yet.", dp.Name)
	return fmt.Errorf("deployment %s not ready", dp.Name)
}

log.Infof("The deployment %s is ready.", dp.Name)
```